src/components/FeedbackButton/FeedbackButton.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+import IconBase from 'react-icons-kit';
+import { messageSquare } from 'react-icons-kit/feather/messageSquare';
+import { Spring, animated, interpolate } from 'react-spring';
+
+import DetectActive from '../DetectActive';
+
+import { openWindow } from '../../services/shell.service';
+import { IN_APP_FEEDBACK_URL } from '../../config/app';
+import { COLORS } from '../../constants';
+
+type Props = {
+  size: number,
+  color: string,
+  hoverColor: string,
+};
+
+class FeedbackButton extends PureComponent<Props> {
+  static defaultProps = {
+    size: 36,
+    color: COLORS.gray[400],
+    hoverColor: COLORS.purple[500],
+  };
+
+  render() {
+    const { color, hoverColor, size } = this.props;
+    return (
+      <DetectActiveWrapper>
+        {(_, isHovered) => (
+          <Spring
+            native
+            to={{
+              scale: isHovered ? 1.15 : 1,
+            }}
+          >
+            {({ scale }) => (
+              <IconWrapper scale={scale} color={isHovered ? hoverColor : color}>

```

Comment:
```suggestion
              <IconWrapper scale={scale} color={isHovered ? hoverColor : color} onClick={() => openWindow(IN_APP_FEEDBACK_URL)}>
```

(Part of overall UI suggestion - remove onClick from `<IconBase>` and put here instead)